addons/website_event/controllers/main.py

```diff
@@ -92,7 +92,7 @@ def sd(date):
             domain_search["country"] = [("country_id", "=", False)]
 
         def dom_without(without):
-            domain = [('state', "in", ['draft', 'confirm', 'done'])]
+            domain = []
```

Comment:
I don't think you want to see canceled events in this kind of controllers ... makes no sense. I would use pipe_cancel here.